tracer/src/Datadog.Trace/DataStreamsMonitoring/DataStreamsManager.cs

```diff
@@ -229,17 +232,4 @@ public void InjectPathwayContextAsBase64String<TCarrier>(PathwayContext? context
             return null;
         }
     }
-
-    private class CheckpointInfo
```

Comment:
this existed to save the context and the direction together. Since we only save the context on consume now, we don't need to know the direction (it's always "in")